src/workloads/scale/MemoryStress.yml

```diff
@@ -0,0 +1,90 @@
+SchemaVersion: 2018-07-01
+Owner: Service Architecture
+Description: |
+  Used to test performance of the server under heavy memory pressure.
```

Comment:
done! left out language related to type of exception, because the actual exception genny encounters is "generic server error" due to "Could not find host matching read preference" (which is a result of mongods crashing from OOM). Added a sentence about how expectation of failure is temporary. LMK if language should be changed in any way